src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -84,11 +84,13 @@ export const Map: React.FC<MapProps> = ({
   const takeSeat = useCallback(
     (row: number | null, column: number | null) => {
       if (!userUid) return;
+
       makeUpdateUserGridLocation({
         venueId,
         userUid,
       })(row, column);
-      trackLocationEntered({ userId: userUid, locationName: venueName });
+
+      setNewLocationData({ userId: userUid, locationName: venueName });
```

Comment:
I kind of feel like the old naming of this (`trackLocationEntered`) was better.

Reply:
This naming aligns with the other function names `updateCurrentLocation`, `clearLocation`

Reply:
Makes sense